src/config/config.ts

```diff
@@ -751,6 +751,7 @@ export async function toCached(c: Command.Class, plugin?: IPlugin): Promise<Comm
         helpGroup: flag.helpGroup,
         allowNo: flag.allowNo,
         dependsOn: flag.dependsOn,
+        relationships: flag.relationships,
```

Comment:
would we want to use union types to make it not possible to use both relationships and any of the simple ones (dependsOn, exclusive, exactlyOne) so that it doesn't get really nasty?

Reply:
I feel comfortable not putting any guard rails up for that. People can make a mess of their flags if they want

For the sf side, we could always add some linting rules to guide people away from doing too much mixing and matching